integrationTests/relayers/slowTests/common.go

```diff
@@ -148,8 +146,7 @@ func GenerateTestEUROCToken() framework.TestTokenParams {
 			},
 		},
 		ESDTSafeExtraBalance: big.NewInt(100), // extra is just for the fees for the 2 transfers mvx->eth
-		EthTestAddrsExtraBalances: map[string][]*big.Int{
-			// 1st position: extra balance for sender, 2nd position: extra balance for receiver
+		ExtraBalances: map[string]framework.ExtraBalanceHolder{
```

Comment:
still hard to read & understand these conditions because the struct fields are not provided. Why not change this to:
```go
ExtraBalances: map[string]framework.ExtraBalanceHolder{
   alice: {
      SentAmount:     big.NewInt(-5010 - 7010 - 1010),
      ReceivedAmount: big.NewInt(0),
   }

```

Valid for all occurrences. Also please make use of the alice, bob, charlie constants instead of the magic strings